lib/service_check.pm

```diff
@@ -41,7 +43,7 @@ our $default_services = {
     ntp => {
         srv_pkg_name  => 'ntp',
         srv_proc_name => 'ntpd',
-        support_ver   => '12-SP3,12-SP4,12-SP5'
+        support_ver   => '12-SP3,12-SP4'
```

Comment:
how remove of SP5 for ntp related to "extending of apache testing" ?

Reply:
changed, cause weigao submitted a PR and I'm not update the file.